migration/idempotent/017-backend-telemetry-buffer.sql

```diff
@@ -0,0 +1,22 @@
+-- The contents of this file was auto-generated by the pgx extension, but was
```

Comment:
I don't think that this file is necessary, because these definitions will be emitted by PGX.